plugins/amazonq/mynah-ui/package.json

```diff
@@ -12,7 +12,7 @@
         "lintfix": "eslint -c .eslintrc.js --fix --ext .ts ."
     },
     "dependencies": {
-        "@aws/mynah-ui-chat": "npm:@aws/mynah-ui@4.4.2",
```

Comment:
please sync with @dogusata on if this change is safe

https://github.com/aws/aws-toolkit-jetbrains/pull/4262

Reply:
Yeah @dogusata advised to change version to get the parameter for both Q chat events from mynah-UI.